modules/data.remote/R/call_MODIS.R

```diff
@@ -85,41 +86,32 @@ call_MODIS <- function(outfolder = ".", start_date, end_date, lat, lon, size = 0
     }
     
     
-    # reformat output from list/factors to simple str+numeric dataframe
-    data = data.frame(lapply(dat$data, as.character), stringsAsFactors = F)
-    
-    headers = (dat$header)
-    scale = as.numeric(headers$scale)
-    # change lat and lon once we start outputing rasters, the lat and lon will not be the same for all data based on input lat and lon
-    latitude = as.character(rep(headers$latitude, nrow(data)))
-    longitude = as.character(rep(headers$longitude, nrow(data)))
-
     if (band_qc == "")
     {
-      QC = rep("nan", nrow(data))
+      QC = rep("nan", nrow(dat))
```

Comment:
@bailsofhay can you please change "=" to "<-" to be consistent with the code in PEcAn?  This only pertains to objects not function calls.  That is if in the function [e.g. call_MODIS(arg1 = XXX) ] is correct.

Reply:
uggggggggggggggggg.... fine!